interfaces/builtin/dsp.go

```diff
@@ -61,16 +61,17 @@ const ambarellaDspConnectedPlugApparmor = `
 /dev/lens rw,
 
 # various ambarella specific DSP control parameters
-/proc/ambarella/iav rw,
-/proc/ambarella/ambnl rw,
-/proc/ambarella/udc rw,
-/proc/ambarella/clock rw,
+/proc/ambarella/iav r,
+/proc/ambarella/ambnl/** rw,
+/proc/ambarella/udc r,
+/proc/ambarella/clock r,
 /proc/ambarella/dsp_print rw,
-/proc/ambarella/hdmi_edid rw,
-/proc/ambarella/cma rw,
+/proc/ambarella/hdmi_edid r,
+/proc/ambarella/cma r,
```

Comment:
A bunch of those was `rw` before, is the snap consuming this interface capable of working properly if a newer snapd lands and updates the interface or do they ensure that the right versions are in place before snapd is updated?